File: common/src/main/java/net/pl3x/map/palette/Palette.java
package net.pl3x.map.palette;

public class Palette {
    private final int index;
    private final String name;

    public Palette(int index, String name) {
        this.index = index;
        this.name = name;
    }

    public int getIndex() {
        return this.index;
    }

    public String getName() {
        return this.name;
    }
}

File: webmap/src/palette/Palette.ts
export class Palette {
    private readonly _index: string;
    private readonly _value: string;

    constructor(index: string, value: string) {
        this._index = index;
        this._value = value;
    }

    get index(): string {
        return this._index
    }

    get value(): string {
        return this._value
    }
}
